scratch/controller/controller.cpp

```diff
@@ -0,0 +1,127 @@
+#include "../../src/network/network.hpp"
+#include <linux/joystick.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <iostream>
+#include <fcntl.h> 
+#include <unistd.h>
+
+const unsigned short 
+    //Button values
+    A_BUTTON = 0,
+    B_BUTTON = 1,
+    X_BUTTON = 2,
+    Y_BUTTON = 3,
+    LB_BUTTON = 4,
+    RB_BUTTON = 5,
+    BACK_BUTTON = 6,
+    START_BUTTON = 7,
+    HOME_BUTTON = 9,
+    //Axis values
+    JOYSTICK_LEFT_X = 0,
+    JOYSTICK_LEFT_Y = 1,
+    JOYSTICK_RIGHT_X = 3,
+    JOYSTICK_RIGHT_Y = 4,
+    DPAD_X = 6,
+    DPAD_Y = 7,
+    LT_AXIS =2,
```

Comment:
spacing is a bit weird. If you want to keep them in line, keep _all_ of the options here in line 
```suggestion
    LT_AXIS = 2,
```